### DISCUSSION TITLE
Настройки для оператора YOTA

### DISCUSSION THREAD

**Discussion by oxotnika** [dovecoteescapee/ByeDPIAndroid#205] [General]
Всем привет. Поделитесь пожалуйста настройками под мобильного оператора YOTA. Что то перестал ByeDPI Android грузить ютуб от слова совсем((( Спасибо.

**Comment by ThousRubles** [comment]
> -H:telegram.org --proto=udp --pf=443 --fake-data=':\xC2\x00\x00\x00\x01\x14\x2E\xE3\xE3\x5F\x6B\xBB\x23\xA8\xE6\x5D\xA9\x78\x21\xCF\xC2\x72\x4C\x8F\xC4\x5E\x14\x00\x00\x00\x00\xC5\x00\x00\x00\x00\x4C\x00\xA7\x00\x00\x00\x00\x00\x00\x44\x00\x00\x80\x00\x00\x00\x0D\xFC\xFA\x1D\xCD\x73\xBA\x2A\x90\x93\xB3\xEE\xF7\x43\xC5\x85\xDA\xFF\x45\x3C\x00\x00\x00\x00\x00\x00\x7C\x00\x9B\x00\xF6\x00\x00\xDD\x00\x00\x00\x00\x00\x00\x00\x00\x00\x59\xA8\xE4\x00\x00\x00\x00\x00\x00\x00\x00\x7B\x00\x0F\x00\x00\x00\x48\x4E\x00\x00\x00\x06\xF3\x00\x00\x00\x00\xD9\x5A\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00' --udp-fake=10 --auto=none -s1 -o1

Некоторые сайты не грузит, добавляй исключения через "-H:", исключения приложений можно через черный список фильтров приложений добавить.

**Reply by ThousRubles** [reply]
Йота СПб, версия 1.3.9

**Reply by codeabuse** [reply]
Спасибо тебе, мил человек! Работает, ютубчик стал грузить видео!

**Reply by codeabuse** [reply]
Локально на некоторых станциях метро и, видимо, кое-где на земле (зависит от антенны) перестали работать настройки. Вот буквально, рядом с метро и внизу ютуб перестает грузить. Питер.

**Reply by codeabuse** [reply]
Новые рабочие настройки, предыдущий вариант дополнен товарищем с соседней ветки. 
`-H:russia-youtube.txt --proto=udp --pf=443 --fake-data=':\xC2\x00\x00\x00\x01\x14\x2E\xE3\xE3\x5F\x6B\xBB\x23\xA8\xE6\x5D\xA9\x78\x21\xCF\xC2\x72\x4C\x8F\xC4\x5E\x14\x00\x00\x00\x00\xC5\x00\x00\x00\x00\x4C\x00\xA7\x00\x00\x00\x00\x00\x00\x44\x00\x00\x80\x00\x00\x00\x0D\xFC\xFA\x1D\xCD\x73\xBA\x2A\x90\x93\xB3\xEE\xF7\x43\xC5\x85\xDA\xFF\x45\x3C\x00\x00\x00\x00\x00\x00\x7C\x00\x9B\x00\xF6\x00\x00\xDD\x00\x00\x00\x00\x00\x00\x00\x00\x00\x59\xA8\xE4\x00\x00\x00\x00\x00\x00\x00\x00\x7B\x00\x0F\x00\x00\x00\x48\x4E\x00\x00\x00\x06\xF3\x00\x00\x00\x00\xD9\x5A\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00' --udp-fake=10 --auto=none -s0 -o1 -d1 -r1+s -Ar -o1 -At -f-1 -r1+s -As`

Взято отсюда: https://github.com/dovecoteescapee/ByeDPIAndroid/issues/267

russia-youtube.txt лежит в Downloads и содержит имена хостов (каждый на новой строке, без запятых), на которые мы хотим попасть (ютуб, его cdn и прочие ресурсы, которые нам интересны).

**Reply by Diilixside** [reply]
> Новые рабочие настройки, следующий вариант дополнен товарищем с соседней веткой. `-H:russia-youtube.txt --proto=udp --pf=443 --fake-data=':\xC2\x00\x00\x00\x01\x14\x2E\xE3\xE3\x5F\x6B\xBB\x23\xA8\xE6\x5D\xA9\x78\x21\xCF\xC2\x72\x4C\x8F\xC4\x5E\x14\x00\x00\x00\x00\xC5\x00\x00\x00\x00\x4C\x00\xA7\x00\x00\x00\x00\x00\x00\x44\x00\x00\x80\x00\x00\x00\x0D\xFC\xFA\x1D\xCD\x73\xBA\x2A\x90\x93\xB3\xEE\xF7\x43\xC5\x85\xDA\xFF\x45\x3C\x00\x00\x00\x00\x00\x00\x7C\x00\x9B\x00\xF6\x00\x00\xDD\x00\x00\x00\x00\x00\x00\x00\x00\x00\x59\xA8\xE4\x00\x00\x00\x00\x00\x00\x00\x00\x7B\x00\x0F\x00\x00\x00\x48\x4E\x00\x00\x00\x06\xF3\x00\x00\x00\x00\xD9\x5A\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00' --udp-fake=10 --auto=none -s0 -o1 -d1 -r1+s -Ar -o1 -At -f-1 -r1+s -As`
> 
> Взятый отсюда: #267
> 
> russia-youtube.txt лежит в разделе «Загрузки» и содержит имена хостов (каждый на новой строке, без пятых), на которые мы хотим попасть (ютуб, его cdn и другие ресурсы, которые нам интересны).

Спасибо бро. Теперь все рабоатет

**Reply by GGee64** [reply]
> Новые рабочие настройки, предыдущий вариант дополнен товарищем с соседней ветки. `-H:russia-youtube.txt --proto=udp --pf=443 --fake-data=':\xC2\x00\x00\x00\x01\x14\x2E\xE3\xE3\x5F\x6B\xBB\x23\xA8\xE6\x5D\xA9\x78\x21\xCF\xC2\x72\x4C\x8F\xC4\x5E\x14\x00\x00\x00\x00\xC5\x00\x00\x00\x00\x4C\x00\xA7\x00\x00\x00\x00\x00\x00\x44\x00\x00\x80\x00\x00\x00\x0D\xFC\xFA\x1D\xCD\x73\xBA\x2A\x90\x93\xB3\xEE\xF7\x43\xC5\x85\xDA\xFF\x45\x3C\x00\x00\x00\x00\x00\x00\x7C\x00\x9B\x00\xF6\x00\x00\xDD\x00\x00\x00\x00\x00\x00\x00\x00\x00\x59\xA8\xE4\x00\x00\x00\x00\x00\x00\x00\x00\x7B\x00\x0F\x00\x00\x00\x48\x4E\x00\x00\x00\x06\xF3\x00\x00\x00\x00\xD9\x5A\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00' --udp-fake=10 --auto=none -s0 -o1 -d1 -r1+s -Ar -o1 -At -f-1 -r1+s -As`
> 
> Взято отсюда: #267
> 
> russia-youtube.txt лежит в Downloads и содержит имена хостов (каждый на новой строке, без запятых), на которые мы хотим попасть (ютуб, его cdn и прочие ресурсы, которые нам интересны).

Все бы ничего, но дискорд ни в какую не хочет работать. Чего я только ни перепробовал...

**Comment by AmSchegar** [comment]
Спасибо за подсказку, но куда вводить эту портянку?

**Reply by PeepoJinxpupps** [reply]
В командную строку Byedpi
